content-tool/tool/src/java/org/sakaiproject/content/tool/ResourcesHelperAction.java

```diff
@@ -171,6 +171,8 @@
 	/** Tool property to enable Drag and Drop uploads in a per-tool basis */
 	private static final String TOOL_PROP_DRAGNDROP_ENABLED = "content.upload.dragndrop";
 
+	private static final String LABEL_VERSION = "label.version";
```

Comment:
Creating a constant for a string that is only use in one place and is only used as a i18n lookup key makes the code harder to follow. It also doesn't fit with the existing pattern that there is in this file (no i18n lookup keys are constants).